doc/classes/Vector2.xml

```diff
@@ -43,7 +43,8 @@
 			<return type="float">
 			</return>
 			<description>
-				Returns this vector's angle with respect to the X axis, or [code](1, 0)[/code] vector, in radians.
+				Returns this vector's angle with respect to the positive X axis, or [code](1, 0)[/code] vector, in radians.
+				For example, [code]Vector2.RIGHT.angle()[/code] will return zero, [code]Vector2.DOWN.angle()[/code] will return [code]PI/2[/code] (90 degrees), and [code]Vector2(1,-1).angle()[/code] will return [code]-PI/4[/code] (-45 degrees).
```

Comment:
```suggestion
				For example, [code]Vector2.RIGHT.angle()[/code] will return zero, [code]Vector2.DOWN.angle()[/code] will return [code]TAU / 4[/code] (a quarter turn, or 90 degrees), and [code]Vector2(1, -1).angle()[/code] will return [code]-TAU / 8[/code] (a negative eighth turn, or -45 degrees).
```

Proper spacing is important, but aside from that, it would be nice to use TAU in examples, since it usually results in more clear code.

Reply:
TAU is much less known though.
(also the missing spaces are a result of copy-paste, oops...)

Reply:
Still should be `PI / 2` instead of `PI/2`.